pylana/logs.py

```diff
@@ -294,8 +318,12 @@ def upload_event_log_file(self, name: str,
             return self.post('/api/logs/csv-case-attributes-event-semantics',
                          files=files, data=semantics, **kwargs)
 
+    # TODO: Check whether impact and descriptive attributes are optional
```

Comment:
for the upload? yes, should be optional here to

Reply:
I removed the TODO